axis.go

```diff
@@ -571,13 +631,13 @@ func log(x float64) float64 {
 	return math.Log(x)
 }
 
-// formatFloatTick returns a g-formated string representation of v
+//formatFloatTick returns a g-formated string representation of v
 // to the specified precision.
 func formatFloatTick(v float64, prec int) string {
-	return strconv.FormatFloat(floats.Round(v, prec), 'g', displayPrecision, 64)
+	return strconv.FormatFloat(v, 'g', displayPrecision, 64)
```

Comment:
I am a bit unconfortable with `formatFloatTick` depending on a global variable (`displayPrecision`) that can/could be modified from various places.
couldn't we either:
- pass it as an additional argument
- or make `formatFloatTick` a method of some new `int`-like type
?

(same comment for `precisionOf`)

Reply:
This one is the most tricky one. I have changed displayPrecision to prec in the fuction returned. Making it int-type might not be a great idea, as in this case all our 'looking for the right precision' work will be useless, right?

Reply:
As for precisionOf, here adding a new argument might work. Will try to fix that.